jetty9/src/main/docker/Dockerfile

```diff
@@ -14,35 +14,28 @@
 
 FROM ${docker.openjdk.image}
 
-# create jetty group and user
-RUN groupadd -r jetty \
- && useradd -r -g jetty jetty
-
 # Create env vars to identify image
 ENV JETTY_VERSION ${jetty9.version}
 ENV GAE_IMAGE_NAME jetty
 ENV GAE_IMAGE_LABEL ${docker.tag.long}
 
 # Create Jetty Home
-ENV JETTY_HOME /usr/local/jetty
+ENV JETTY_HOME ${jetty.home}
 ENV PATH $JETTY_HOME/bin:$PATH
 ENV TMPDIR /tmp/jetty
-ADD jetty-distribution.tar.gz /usr/local/
-RUN mv /usr/local/jetty-distribution-* $JETTY_HOME \
- && chown -R jetty:jetty $JETTY_HOME
+RUN mkdir $TMPDIR
+ADD jetty-distribution.tar.gz $JETTY_HOME/..
```

Comment:
First off, the jetty-home artifact might be better to use, we started using it in Jetty-9.4 and it is basically the distribution minus the test applications, etc...bit of a smaller footprint.

Secondly, that extra directory is an artifact of it being an archive and no one likes to unpack a zip file and have a mess of files show up in their current directory.  I have tinkered around in the past and never found a clean way of doing what you are asking through Maven.  Short of some antrun after unpack...

  ```
 <copy todir="target/docbkx/html/documentation/9.3.9.v20160517">
    <fileset dir="target/documentation-resource/9.3.9.v20160517/jetty-documentation-9.3.9.v20160517/html"/>
</copy>
```